src/Lucene.Net.ObjectMapping/Lucene.Net.ObjectMapping.csproj

```diff
@@ -31,67 +32,37 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Release\Lucene.Net.ObjectMapping.XML</DocumentationFile>
+    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <PropertyGroup>
-    <SignAssembly>true</SignAssembly>
+    <SignAssembly>false</SignAssembly>
```

Comment:
You mustn't change this -- the assemblies should remain signed for publishing to Nuget